pkg/util/api.go

```diff
@@ -146,7 +146,7 @@ func NewJSONResponse(code int, value interface{}) (*web.Response, error) {
 
 func NewJSONResponseWithHeaders(code int, value interface{}, additionalHeaders map[string]string) (*web.Response, error) {
 	headers := http.Header{}
-	headers.Add("Content-Type", "application/json")
+	headers.Set("Content-Type", "application/json")
```

Comment:
any particular reason to Set and not Add ?

Reply:
My thinking here is, because if this is called multiple times, Set will replace the header, but Add will add another value. For example for cookies you can set multiple, but for Content-Type you shouldn't be able to do this